### DISCUSSION TITLE
No Optimization After logging in a New local User

### DISCUSSION THREAD

**Discussion by Keshav-writes-code** [Atlas-OS/Atlas#1162] [Help & Support] [answered]
# **Steps to Recreate**

- Do a Fresh Install of Windows 11 23H2 (microsoft official ISO)
- Run the Disable Automatic Driver Installation.reg immediately
- avoid any windows updates
- Do the Atlas Os Installation Process
- after, Create new User Settings > Account > Other Users > Add Account
- sign out of current User and log in to the new one
- Notice, no Optimizatino done for the new Account

# **What Used to Happen?**

- So, i tested after a Fresh install, ran the Driver Disable reg file and did the Atlas OS Installation.
- then when i logged into a New User, it asked me a Yes/No Question in Powershell to set up New user and Restart something like that.

# **What might be happening**
- Completely Made up but i think, the Widnows Updates Removed/Modified a Atlas OS Schedule of some sort that runs when you log in to a New Account for the First Time

**Comment by Ast3risk-ops** [comment, marked as answer]
Might be better as an [issue](../../../Atlas-os/Atlas/issues).

**Comment by he3als** [comment]
Closing as it's now an issue
